src/input.rs

```diff
@@ -469,7 +470,7 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             return;
         }
 
-        self.ctx.copy_selection(Buffer::Selection);
+        self.ctx.copy_selection(self.selection_buffer.clone());
```

Comment:
If `Copy` is also derived on the `Buffer` enum, this `.clone()` call can be removed.